Kudos/Bot/Honor.cs

```diff
@@ -68,7 +90,24 @@ public sealed class Honor {
 
 		public void SendHonorBalance(SocketUser user, ISocketMessageChannel channel) {
 			int honor = BalancesPerId.ContainsKey(user.Id) ? BalancesPerId[user.Id] : 0;
-			Messaging.Instance.Message(channel, $"{user.Mention} has {honor} honor points.");
+			string honorMessage;
+			int index = Program.Random.Next(0, HonorFeedbackLowest.Length);
```

Comment:
dont use same index for everything (could be to high if HonorFeedbackLowest has more entries than sth else)